api/models.py

```diff
@@ -65,6 +66,23 @@ def __repr__(self):
         return f"{self.username}"
 
 
+class Comment(BaseModel):
+    __tablename__ = "comment"
+    to_bot = db.Column(
```

Comment:
if that's and id add an _id suffix. otherwise one can expect it to be actual bot object